riscv/csrs.cc

```diff
@@ -497,6 +490,20 @@ bool mstatus_csr_t::unlogged_write(const reg_t val) noexcept {
   return true;
 }
 
+reg_t mstatus_csr_t::compute_mstatus_initial_value() const noexcept {
+#ifdef RISCV_ENABLE_DUAL_ENDIAN
```

Comment:
I don't the either of these `#ifdef`s are needed here. `is_target_big_endian()` will always be false if this macro is undefined, and it doesn't hurt to always calculate these values.

Reply:
Yeah, I think so and remove the similar  #ifdef RISCV_ENABLE_DUAL_ENDIAN in other functions.